### PR TITLE
Save db

### LINKED ISSUE #3: Weeks 3-4
- [x] Create a form (no CSS yet) that requires the person's email, name and some other fields
- [x] Post the results back to a POST endpoint in the server
- [x] Use [express body parser](https://github.com/expressjs/body-parser) with `Content-Type of x-www-form-urlencoded` to parse the response from `req.body` on the server
- [x] console.log the user responses
- [x] Setup mongoDB locally
- [x] Read up on Mongo and Mongoose
- [x] Setup mongoDB in Heroku using mLab
- [x] Save responses to database
    - [x] Create a Responses Model object
    - [x] Make sure each each response stored is unique, using the person's email as the unique id (via Mongoose)

### PR DESCRIPTION
**Pull request by j-mes**
Closes #3.

This adds an event where if the email address is in the database already, and redirects to form-error which will show the reason and to offer the user the option to go back to the form to try a different email address.

### CODE REVIEW

File: server/controllers/post-form-controller.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const main = (req, res) => {
 	const { name, email, country, gdpr } = req.body;
-	console.log('name:', name);
-	console.log('email:', email);
-	console.log('country:', country);
-	console.log('gdpr:', gdpr);
+	// console.log('name:', name);
+	// console.log('email:', email);
+	// console.log('country:', country);
+	// console.log('gdpr:', gdpr);
```
Comment (lc512k) [inline server/controllers/post-form-controller.js]: do we need this commented code?

**Review by lc512k — APPROVED** [review]
Great work!

File: server/controllers/post-form-controller.js (hunk 1 above)
Comment (j-mes) [reply server/controllers/post-form-controller.js]: Not really, it was only there to show what values were being used. I'll remove it now.

**Comment by lc512k** [comment]
go for it! 👍

**Pull request merged**